yt_dlp/extractor/newgrounds.py

```diff
@@ -4,54 +4,81 @@
 
 from .common import InfoExtractor
 from ..utils import (
-    ExtractorError,
     extract_attributes,
     int_or_none,
     parse_duration,
     parse_filesize,
+    parse_count,
     unified_timestamp,
 )
 
 
 class NewgroundsIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?newgrounds\.com/(?:audio/listen|portal/view)/(?P<id>[0-9]+)'
+    _VALID_URL = r'https?://(?:www\.)?newgrounds\.com/(?:audio/listen|portal/view)/(?P<id>\d+)(?:/format/flash)?'
     _TESTS = [{
         'url': 'https://www.newgrounds.com/audio/listen/549479',
         'md5': 'fe6033d297591288fa1c1f780386f07a',
         'info_dict': {
             'id': '549479',
             'ext': 'mp3',
-            'title': 'Burn7 - B7 - BusMode',
+            'title': 'B7 - BusMode',
             'uploader': 'Burn7',
             'timestamp': 1378878540,
             'upload_date': '20130911',
             'duration': 143,
+            'description': 'Sonic blahblahblah I\'m late again SEPTEMBER 11TH: Busmode-The story of ants on a log. ',
```

Comment:
nothing wrong with using `\'` but you can also use md5.
paste the text here `https://www.md5hashgenerator.com/` and then add it as 
`'description': 'md5:<md5hash>',`